mysqld_exporter.go

```diff
@@ -726,6 +729,14 @@ func NewExporter(dsn string) *Exporter {
 			Name:      "scrapes_total",
 			Help:      "Total number of times MySQL was scraped for metrics.",
 		}),
+		scrapeErrors: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: exporter,
+			Name:      "scrape_errors_total",
+			Help:      "Total number of times an error occured scraping a MySQL.",
+		},
```

Comment:
I did it this way because i was getting compiler errors because of other bugs. I though it was my labels literal that was causing problems.